extensions/roc-package-web-app-react/app/client/create-client.js

```diff
@@ -186,5 +186,31 @@ export default function createClient({ createRoutes, createStore, mountNode }) {
         }
     };
 
-    render();
+    if (USE_I18N_POLYFILL) {
+        let intlLoader;
```

Comment:
This is really a minor thing and does not really matter but I would use a ternary here to make the definition closer to the assignment, what do you think?

```js
const intlLoader = !global.Intl ?
  require('bundle?name=intl!intl') :
  (cb) => cb();
```